cli/src/test/java/org/jboss/gm/cli/DifferentJVMTest.java

```diff
@@ -291,19 +294,20 @@ public void runWithJDK8Fails() throws Exception {
         try {
             m.run(args);
         } catch (Exception e) {
-            if (v.compareTo(GradleVersion.version("5.0")) >= 0) {
-                // XXX: Ignored for 4.10 as Exception differs between Gradle versions and System.err is checked later anyway
+            if (v.compareTo(GradleVersion.version("5.0")) >= 0 && v.compareTo(GradleVersion.version("7.2")) < 0) {
+                // XXX: Ignored for 4.10 and 7.2+ as Exception differs between Gradle versions and System.err is
+                // XXX: checked later anyway
                 assertTrue(e.getMessage().contains("Problem executing build"));
             }
         }
 
-        File gmeGradle = new File(projectRoot.getParentFile().getAbsolutePath(), "gme.gradle");
+        File gmeGradle = new File(projectRoot.getParentFile().getAbsolutePath(), AlignmentTask.GME);
         assertFalse(gmeGradle.exists());
         // XXX: Caused by: java.io.StreamCorruptedException: invalid type code: 00
         assumeTrue(v.compareTo(GradleVersion.version("5.3.1")) != 0);
         assertTrue(systemErrRule.getLog().contains("Task 'FOOgenerateAlignmentMetadataBAR' not found in root project"));
         // XXX: Gradle 4.x doesn't trigger the same Exception
-        assumeTrue(v.compareTo(GradleVersion.version("5.0")) >= 0);
+        assumeTrue(v.compareTo(GradleVersion.version("5.0")) >= 0 && v.compareTo(GradleVersion.version("7.2")) < 0);
```

Comment:
Why are we ignoring Gradle versions > 7.1 ?

Reply:
It doesn't pass this check in 7.2 (same as in 4.10) because the same Exception is not thrown. I suppose I should have changed the comment as well.

Reply:
Does this mean it wouldn't 'pass' the `assertTrue(systemOutRule.getLog().contains("Java home overridden to: " + JDK8_DIR));` ? 
Frankly the Gradle constant change of their exception handling is a pain :-(

Reply:
I think this then stops the final assert working - see my other comment.